### DISCUSSION TITLE
Is it possible to get vital signs details for subject_ids that did not get to ICU

### DISCUSSION THREAD

**Discussion by emmaxcojil** [MIT-LCP/mimic-code#1590] [MIMIC-IV]
Please i will be truly glad if anyone can assist as the chartevents table of the mimic iv version 2.2 only contains labels/vital signs for subject who were admitted to ICU. I have been looking for ways to get vital signs like (Heart Rate, Respiratory Rate, O2 saturation pulseoxymetry, Heart Rhythm etc) for subjects not admitted to ICU (that is subject_ids that were admitted but did not progress to any of the ICU's)

**Comment by AntonQUT** [comment]
I have the same question. There are vital signs for ICU patients and ED patients, but not ward patients, yet ward patients always have their vitals checked.  Is there anyway to access this information for these patients?

**Comment by alistairewj** [comment]
Not at the moment, because as far as I understand most ward patients had vitals collected on paper.
